### PR TITLE
Bug 1173675 - link gaia jsdoc gh-page on per app readme

### PR DESCRIPTION
**Pull request by gasolin**
…iros, mcavanaugh, jburke, rudy, jporter, steve

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1c0b239d9a8308f76987f2774160c4ecdec3ee4f

File: apps/keyboard/README.md [hunk 1]
```diff
@@ -14,3 +14,11 @@ TEST_FILES=apps/keyboard/test/build/integration/keyboard_test.js make build-test
 for build test.
 
 If you change the IME files, the dictionary filesize will be changed as well. Remember to update the test config `dictFileSize` in `apps/keyboard/test/build/integration/resources/`.
+
+## JSDOC
+
+Generated jsdoc is hosted in [http://mozilla-b2g.github.io/gaia/keyboard/](http://mozilla-b2g.github.io/gaia/keyboard/). You can generate it locally with following command:
```
Comment (RudyLu) [inline apps/keyboard/README.md]: hosted `on`?
with `the` following comand.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3166e08032b32ffa8ba8a22091de870b751172e6

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b520db0e49522bcb311ed3df569744cbf6e1888d

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6dda3fbb0aa0511c43e8da365660dac41de90435

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3148423484fe4917605e41a17a1e38f202b60538

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6c53e2e4745ae411e186d9c2c7f064b55750caf4

**Pull request merged**